devices/Ssd13xx/Ssd1306.cs

```diff
@@ -28,14 +36,95 @@ public Ssd1306(I2cDevice i2cDevice) : base(i2cDevice)
+        /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
+        /// light emitting diode dot-matrix graphic display system.
+        /// </summary>
+        /// <param name="i2cDevice">The I2C device used for communication.</param>
+        /// <param name="res">Display resolution</param>
+        public Ssd1306(I2cDevice i2cDevice, DisplayResolution res) : base(i2cDevice,res)
+        {
+        }
+
         /// <summary>
         /// Initializes new instance of Ssd1306 device that will communicate using I2C bus.
         /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
         /// light emitting diode dot-matrix graphic display system.
         /// </summary>
         /// <param name="i2cDevice">The I2C device used for communication.</param>
         /// <param name="res">Display resolution</param>
-        public Ssd1306(I2cDevice i2cDevice, DisplayResolution res) : base(i2cDevice, res)
+        /// <param name="resetPin">Reset pin (some displays might be wired to share the microcontroller's
+        /// reset pin).</param>
+        public Ssd1306(I2cDevice i2cDevice, DisplayResolution res, int resetPin) : base(i2cDevice, res,resetPin)
+        {            
+        }
+
+        /// <summary>
+        /// Initializes new instance of Ssd1306 device that will communicate using I2C bus.
+        /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
+        /// light emitting diode dot-matrix graphic display system.
+        /// </summary>
+        /// <param name="i2cDevice">The I2C device used for communication.</param>
+        /// <param name="res">Display resolution</param>
+        /// <param name="resetPin">Reset pin (some displays might be wired to share the microcontroller's
+        /// reset pin).</param>
+        /// <param name="gpio">Gpio Controller.</param>          
+        public Ssd1306(I2cDevice i2cDevice, DisplayResolution res, int resetPin, GpioController gpio) : base(i2cDevice,res, resetPin,gpio)
+        {
+        }
+
+        /// <summary>
+        /// Initializes new instance of Ssd1306 device that will communicate using I2C bus.
+        /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
+        /// light emitting diode dot-matrix graphic display system.
+        /// </summary>
+        /// <param name="i2cDevice">The I2C device used for communication.</param>
+        /// <param name="res">Display resolution</param>
+        /// <param name="resetPin">Reset pin (some displays might be wired to share the microcontroller's
+        /// reset pin).</param>
+        /// <param name="gpio">Gpio Controller.</param> 
+        /// <param name="shouldDispose">True to dispose the GpioController.</param>        
+        public Ssd1306(I2cDevice i2cDevice, DisplayResolution res, int resetPin, GpioController gpio, bool shouldDispose) : base(i2cDevice, res, resetPin, gpio, shouldDispose)
```

Comment:
you just need this constructor with default values for the GpioController and the shouldDispose, you can then remove all the others, it will simplify a bit the code.